CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.4.0] - 2017-07-25
```

Comment:
This should be today's date :)

Reply:
It was bound to happen

Reply:
Ah, I knew I would forget something! Thanks!